docs/hugo/content/introduction/multitenant-deployment.md

```diff
@@ -0,0 +1,68 @@
+# Deploying ASO v2 in multi-tenant mode
+
+The current release version of the ASO deployment YAML installs the operator in single-tenant mode:
+it's deployed in the `azureserviceoperator-system` namespace with one set of Azure credentials, and manages resources in any namespace in the cluster.
+That single operator deployment handles webhooks fired when Azure resources are changed.
+
+ASO can also be deployed in a multi-tenant configuration that enables using separate credentials to manage resources in different Kubernetes namespaces.
+
+Running the operator in multi-tenant mode requires one deployment to handle webhooks (since webhook configurations are cluster-level resources) and then a deployment for each tenant, each with its own credentials and set of namespaces that it watches for Azure resources.
+To deploy the operator in multi-tenant mode the release YAML has been split into two parts:
+
+1. Cluster-wide resources:
+   * Custom resource definitions for the Azure resources
+   * Cluster roles for managing those resources
+   * The `azureserviceoperator-system` namespace containing the deployment and service to run the operator.
```

Comment:
A bit confused about this. In multitenant mode this namespace would just be for the webhook deployment, since it's running in webhook only mode, right?

Reply:
Clarified this (in conjunction with the comment below).